source/database-connections/manage-database-connections.txt

```diff
@@ -0,0 +1,86 @@
+.. _rm-manage-connections:
+
+===========================
+Manage Database Connections
+===========================
+
```

Comment:
(Information typing)
These are three tasks. I know they're super little, but It would be a good idea to handle our information typing style consistently across the repo.

Reply:
Discussed over slack. We can come up with a unified approach a little later.